test/unit/ajax-test.js

```diff
@@ -135,7 +135,7 @@ define([
         var done = assert.async();
 
         utils.ajax({
-            url: '//playertest.longtailvideo.com/vast/preroll.xml',
+            url: require.toUrl('./data/playlist.xml'),
```

Comment:
Make sure this works when running tests in the browser (open test/qunit/) and in karma.